### PR TITLE
[scudo] Make comment compatible with gcc

### PR DESCRIPTION
**Pull request by Caslyn**
gcc interprets a backslash '\\' as the last char before a new line as a line continuation character, even in a comment context. This can produce an "error: multi-line comment [-Werror=comment]".

This PR places an innocuous '.' following the '\' so that the comment can compile with gcc.

### CODE REVIEW

**Review by fmayer — APPROVED** [review]
LGTM

alternatively we could change it to `|` on both ends. no preference on my end.

File: compiler-rt/lib/scudo/standalone/secondary.h [hunk 1]
```diff
@@ -426,7 +426,7 @@ class MapAllocatorCache {
     //  The difference between the retrieved memory chunk and the request
     //  size is at most MaxAllowedFragmentedPages
     //
-    //  / MaxAllowedFragmentedPages * PageSize \
+    //  / MaxAllowedFragmentedPages * PageSize \.
```
Comment (ChiaHungDuan) [inline compiler-rt/lib/scudo/standalone/secondary.h:429]: Can we change it to `+-` and `-+` like https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/scudo/standalone/primary64.h#L102 ?

Comment (Caslyn) [reply compiler-rt/lib/scudo/standalone/secondary.h]: GTK, done.

**Review by ChiaHungDuan — APPROVED** [review]

**Comment by Caslyn** [comment]
Thanks @fmayer and @ChiaHungDuan for the review - moving forward with merging.

**Pull request merged**